tempto-examples/src/main/resources/sql-tests/datasets/sample_jdbc_table.ddl

```diff
@@ -1,5 +1,5 @@
 -- type: jdbc
 CREATE TABLE %NAME% (
-  id INT,
+  id BIGINT,
```

Comment:
it is related I used this table to insert data for with presto jdbc table manager